.buildkite/pipeline.sh

```diff
@@ -2,6 +2,9 @@
 
 set -eu
 
+echo "agents:"
+echo '  queue: "public"'
```

Comment:
your intermingling of ' and " is low energy

Reply:
Fixed it, double quotes for all.